package.json

```diff
@@ -80,6 +80,6 @@
 		"tslint-config-prettier": "^1.18.0",
 		"tslint-plugin-prettier": "^2.3.0",
 		"typescript": "^4.0.3",
-		"uglifyjs": "^2.4.11"
+		"uglify-js": "^3.12.1"
```

Comment:
any specific reason to upgrade this?

Reply:
although it's always good to upgrade libraries, was just curious

Reply:
It's been deprecated: https://www.npmjs.com/package/uglifyjs